plugins/reach/lib/api/data/entryVendorTaks/KalturaVendorTaskData.php

```diff
@@ -26,4 +30,13 @@ public static function getInstance($sourceObject, KalturaDetachedResponseProfile
 		
 		return $taskData;
 	}
+
+	protected function validateCaptionAsset($captionAssetId)
```

Comment:
Since caption asset is a property of child KalturaTranslationVendorTaskData., no use add this function here. Move to the child.